openvpncloud/resource_network.go

```diff
@@ -336,25 +336,3 @@ func resourceNetworkDelete(ctx context.Context, d *schema.ResourceData, m interf
 	}
 	return diags
 }
-
```

Comment:
Was this function not used? I may have copy/pasted something at some point and left it there inadvertently 🙂

Reply:
Yep, you can `grep` or search it using some other means. `golangci-lint` also les me that it's redundant.